src/commands/react.ts

```diff
@@ -0,0 +1,168 @@
+
+import * as Discord from 'discord.js';
+
+import { Inject, AutoWired, Singleton } from 'typescript-ioc';
+
+import { ICommand, ICommandArgs, ICommandResult } from '../interfaces';
+import { EmojiService } from '../services/EmojiService';
+import { Logger } from '../services/Logger';
+import { ReactService } from '../services/ReactService';
+import { ErrorMessageService } from '../services/ErrorMessageService';
+import { Channel, DMChannel, GroupDMChannel, Message, TextChannel } from 'discord.js';
+
+@Singleton
+@AutoWired
+export class ReactCommand implements ICommand {
+
+  help = 'React to the message immediately above the command or react to a specific' +
```

Comment:
Use a template string (`) instead of this.

Reply:
- [ ] Use a template string (`) instead of this.